src/data/dialogmotekandidat/types/dialogmoteunntakTypes.ts

```diff
@@ -0,0 +1,12 @@
+export enum UnntakArsak {
+  MEDISINSKE_GRUNNER = "MEDISINSKE_GRUNNER",
+  INNLEGGELSE_INSTITUSJON = "INNLEGGELSE_INSTITUSJON",
+  FORVENTET_FRISKMELDING_INNEN_28UKER = "FORVENTET_FRISKMELDING_INNEN_28UKER",
+  DOKUMENTERT_TILTAK_FRISKMELDING = "DOKUMENTERT_TILTAK_FRISKMELDING",
+  ARBEIDSFORHOLD_OPPHORT = "ARBEIDSFORHOLD_OPPHORT",
+}
+
+export interface NewUnntakDTO {
```

Comment:
Jeg er ikke noe fan av å kalle ting `New`, må den hete det?

Reply:
Det blir vel fort en UnntakDTO (med noen ekstra felter) her senere når vi skal hente med med GET for å vise i historikken. Så vi må kunne skille på de i hvert fall.

Reply:
Har du et annet forslag til navn som betyr nytt-unntak?

Reply:
Det man sender ut vil typisk ha created-timestamp f.eks. som blir generert når det lagres i backend. Kan være de "tilfeldigvis" blir like hvis vi ikke skal bruke created til noe i frontend, men jeg synes det er ryddige å ha to typer for det uansett.

Reply:
Er forskjellen på objekter inn og ut at de som kommer inn til frontend er persistert (og dermed har en createdAt)? Kan det i så fall brukes til å navngi disse?

Reply:
CreateUnntakDto, SettUnntakDto, PostUnntakDto er andre muligheter (liker nok det første best). Så er det tydeligere at typen definerer det som kreves for å lage et unntak.

Reply:
Jeg synes `create` bedre forklarer den forskjellen Erik skrev om - siden "nytt" og "nytt" er samme ord, så det er ikke så lett å formidle hvilken betydning man mener. (Men da blir det et verb, og det er kanskje dumt 😅 )
Men jeg lurer fortsatt på hvorfor man må ha forskjellige klasser for nye og eksisterende unntak 🤷‍♂️

Reply:
Da tar jeg og endrer til Create inntil videre.